test/base-warnings.js

```diff
@@ -0,0 +1,11 @@
+var tape = require('tape');
+var mjAPI = require("../lib/main.js");
+mjAPI.config({undefinedCharError: true});
+
+tape('basic test: check warnings', function (t) {
+    t.plan(2);
+    mjAPI.typeset({math:'呵', html:true})
+        .catch(errors => t.ok(errors, 'CommonHTML output reports error'));
+    mjAPI.typeset({math:'呵', svg:true})
```

Comment:
One might argue that this won't be a problem on engines that understand promises but I'm happy to make the change.